app/errors.js

```diff
@@ -8,3 +8,9 @@ exports.databaseError = message => internalError(message, exports.DATABASE_ERROR
 
 exports.DEFAULT_ERROR = 'default_error';
 exports.defaultError = message => internalError(message, exports.DEFAULT_ERROR);
+
+exports.VALIDATION_ERROR = 'validation_error';
+exports.validationError = message => internalError(message, exports.VALIDATION_ERROR);
+
+exports.MAIL_EXIST_ERROR = 'validation_error';
+exports.mailExistError = message => internalError(message, exports.VALIDATION_ERROR);
```

Comment:
here you are passing a wrong `internalCode` because this error is for existing mails